mpl/comm_group.hpp

```diff
@@ -3533,7 +3533,7 @@ namespace mpl {
       return MPI_SUCCESS;
     }
 
-    static int ialltoallv_cancel(void *state, int complete) { return MPI_SUCCESS; }
+    static int ialltoallv_cancel([[maybe_unused]] void *state, [[maybe_unused]] int complete) { return MPI_SUCCESS; }
```

Comment:
Ok.`ialltoallv_cancel` is intentionally a no-op. Parameters are technically required but intentionally not used.